HBN/dictionary.py

```diff
@@ -0,0 +1,198 @@
+#! /usr/bin/env python3
+
+
+# imports
+import json
+import os
+import pandas
+from pathlib import Path
+
+# file path handling
+HERE = Path(__file__).parent.resolve()
+INPUT_DIR = HERE.joinpath('data_dictionaries')
```

Comment:
```suggestion
INPUT_DIR = HERE.joinpath('Data Dictionaries')
```

For the same reason above in the README.md comment.